executor/test_linux.h

```diff
@@ -263,3 +278,30 @@ static void dump_cpu_state(int cpufd, char* vm_mem)
 	printf(" SRR0 %016lx  SRR1 %016lx\n", regs.srr0, regs.srr1);
 #endif
 }
+
+#ifdef GOARCH_amd64
+// retcodes:
+//  0 : feature disabled
+//  1 : feature enabled
+// -1 : error getting feature state
+static int cpu_feature_enabled(uint32_t function, uint32_t eax_bits, uint32_t ebx_bits, uint32_t ecx_bits, uint32_t edx_bits)
+{
+	int kvmfd = open("/dev/kvm", O_RDWR);
+	if (kvmfd == -1) {
+		perror("Error opening /dev/kvm");
```

Comment:
Nit: we don't use `perror()` anywhere else, let's just use printf for consistency.